test/extensions/path/uri_template_lib/uri_template_test.cc

```diff
@@ -235,59 +224,36 @@ TEST_P(RewritePathTemplateSuccess, RewritePathTemplateSuccessTest) {
 }
 
 TEST(RewritePathTemplateFailure, BadRegex) {
-  envoy::extensions::uri_template::RewriteSegments rewrite_proto;
-
-  const std::string yaml = R"EOF(
-segments:
-- literal: "/"
-- capture_index: 1
-  )EOF";
-
-  Envoy::TestUtility::loadFromYaml(yaml, rewrite_proto);
+  RewriteSegments rewrite_proto;
```

Comment:
here and elsewhere "proto" => "segments", I think

Reply:
changed